### PR TITLE
documentation for dkg create

### PR DESCRIPTION
**Pull request by patnir**
### What changed?

- Adds `wallet:multisig:dkg:create` documentation

---

<sub>**Reminder:** If content (docs, blogs, pages) is moved or renamed, please ensure there are no broken links.</sub>

### CODE REVIEW

File: content/get-started/cli-cmd-wallet-multisig-dkg-create.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+Collecting Participant Info and Performing Round 1...
+Identity for ledger:
+72b58c3e40bb489462b998bfb700b165bb0e243c71744061fb928ffd2e0a2c6c44a587279088fe3f672b57e509e6cc7eba26a50384a1cd21a425d6a73d5e00e90a3d84c4f846183a06773dc63cea2c99aa54c8ef78b87963574268c4d93a26cd1ce800cf97fb5494992ee2893aa798730b3a1fa5a64936a5d82ad7739bcf728a08
+
+
+Enter 1 identities of all other participants (excluding yours)
+Participant Identity #1: 72f9ec850199e49d1da7b95464136a6fea86970b0be2a3f4c9d910fce8791c7cfdd602bde9b38a45e9f03b34a732a2118a1b84edff85bdb2b8edb4ead7b47bae40cb2699c7102ac67050f89a3751e5eaa635386b9c5033bad03b2fd1ec65c606c4408d0c228c4fa9de60b727d161b210649fa39d3f9da23772608e857b91427b0f
+Round1 on Ledger... Ledger Locked
+? Ledger Locked. Unlock and press enter to retry: Retry
```
Comment (hughy) [inline content/get-started/cli-cmd-wallet-multisig-dkg-create.mdx:43]: Do we want to keep the retries in the example?

File: content/get-started/cli-cmd-wallet-multisig-dkg-create.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+Performs the entire process of creating a multisignature account using the Distributed Key Generation (DKG) protocol.
+
+This CLI command is a shortcut for running the following commands in sequence:
+
+1. [`wallet:multisig:dkg:round1`](/use/get-started/cli-cmd-wallet-multisig-dkg-round1)
+2. [`wallet:multisig:dkg:round2`](/use/get-started/cli-cmd-wallet-multisig-dkg-round2)
+3. [`wallet:multisig:dkg:round3`](/use/get-started/cli-cmd-wallet-multisig-dkg-round3)
+
+This command has built in retry logic to handle failures and will not create the account until the DKG process is successfully run.
```
Comment (hughy) [inline content/get-started/cli-cmd-wallet-multisig-dkg-create.mdx:14]:
```suggestion
This command has built-in retry logic to handle failures and will not create the account until the DKG process is successfully run.
```

File: content/get-started/cli-cmd-wallet-multisig-dkg-create.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+Further more, a primary benefit of this command is when using the `--ledger` flag, the command will retrieve the identities from the Ledger device and handle retries/ failures for you.
```
Comment (hughy) [inline content/get-started/cli-cmd-wallet-multisig-dkg-create.mdx:16]:
```suggestion
Furthermore, a primary benefit of this command is when using the `--ledger` flag, the command will retrieve the identities from the Ledger device and handle retries/failures for you.
```

**Review by hughy — APPROVED** [review]

File: content/get-started/cli-cmd-wallet-multisig-dkg-create.mdx (hunk 1 above)
Comment (patnir) [reply content/get-started/cli-cmd-wallet-multisig-dkg-create.mdx:43]: I'll remove it

**Pull request merged**